sdk/app_cpu1/common/sys/scheduler.c

```diff
@@ -23,18 +23,19 @@ static volatile uint32_t elapsed_usec = 0;
 static bool tasks_running = false;
 static volatile bool scheduler_idle = false;
 
-void scheduler_timer_isr(void *arg)
+void scheduler_tick(void)
 {
 #if USER_CONFIG_ENABLE_TIME_QUANTUM_CHECKING == 1
     // We should be done running tasks in a time slice before this fires,
     // so if tasks are still running, we consumed too many cycles per slice
     if (tasks_running) {
-        // Per AMDC-Firmware Issue #265, during start-up, the overrun check
-        // seems to falsely trigger, or at least, we do not care about the trigger.
+        // When enabling the first sensor, the overrun check will occur
```

Comment:
"For the first callback after enabling sensors in the Timing Manager, the overrun check..."

Reply:
I was trying to think of a good way to word this - thanks!